src/main/kotlin/com/terraformation/backend/report/db/ReportStore.kt

```diff
@@ -380,6 +384,39 @@ class ReportStore(
     }
   }
 
+  /** Save the seed bank buildStartDate, buildCompletedDate, and operationStartDate */
+  private fun saveSeedBankInfo(body: ReportBodyModel) {
+    val reportBody = body.toLatestVersion()
+    reportBody.seedBanks
+        .filter { it.selected }
+        .forEach {
+          dslContext
+              .update(FACILITIES)
+              .set(FACILITIES.BUILD_STARTED_DATE, it.buildStartedDate)
+              .set(FACILITIES.BUILD_COMPLETED_DATE, it.buildCompletedDate)
+              .set(FACILITIES.OPERATION_STARTED_DATE, it.operationStartedDate)
```

Comment:
Add a where clause checking facilities type is SeedBank? I understand the id check may be sufficient. Same for the nursery info below.